Okta.Xamarin/Okta.Xamarin/OktaContext.cs

```diff
@@ -185,14 +258,66 @@ public virtual async Task SignOutAsync(IOidcClient oidcClient = null)
         /// </summary>
         /// <param name="tokenType">The type of token to revoke</param>
         /// <returns>Task</returns>
-        public virtual async Task RevokeTokenAsync(TokenType tokenType)
+        public virtual async Task RevokeTokenAsync(TokenKind tokenType)
```

Comment:
It's probably worth renaming the parameter as well: `tokenType` -> `tokenKind`

Reply:
Good catch.  I'll update shortly.